### PR TITLE
Modifications for 5.0 release

### PR DESCRIPTION
**Pull request by ilovesoup**
### What is changed, added or deleted? (Required)

Modifications for 5.0 on TiFlash.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5902
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@ilovesoup Could you invite a technical review? Thanks~

File: tiflash/use-tiflash.md [hunk 1]
```diff
@@ -277,11 +277,20 @@ explain select count(*) from customer c join nation n on c.c_nationkey=n.n_natio
...
-> By default, TiFlash does not support using the MPP mode in partitioned tables.
-
 TiFlash provides the following two global/session variables to control whether to use Broadcast Hash Join:
 
 - [`tidb_broadcast_join_threshold_size`](/system-variables.md#tidb_broadcast_join_threshold_count-new-in-v50-ga): The unit of the value is bytes. If the table size (in the unit of bytes) is less than the value of the variable, the Broadcast Hash Join algorithm is used. Otherwise, the Shuffled Hash Join algorithm is used.
 - [`tidb_broadcast_join_threshold_count`](/system-variables.md#tidb_broadcast_join_threshold_count-new-in-v50-ga): The unit of the value is rows. If the objects of the join operation belong to a subquery, the optimizer cannot estimate the size of the subquery result set, so the size is determined by the number of rows in the result set. If the estimated number of rows in the subquery is less than the value of this variable, the Broadcast Hash Join algorithm is used. Otherwise, the Shuffled Hash Join algorithm is used.
+
+> **Note:**
+>
+> TiFlash Computation layer does not support:
```
Comment (tisonkun) [inline tiflash/use-tiflash.md:287]:
```suggestion
> TiFlash computation layer does not support:
```

or just TiFlash doesn't support? I don't think a user is interested with what "computation layer" is.

File: tiflash/use-tiflash.md [hunk 2]
```diff
@@ -277,11 +277,20 @@ explain select count(*) from customer c join nation n on c.c_nationkey=n.n_natio
...
+>
+> - Overflow check. For example, adding 2 maximum value of `bigint` type `9223372036854775807 + 9223372036854775807` results in error message `ERROR 1690 (22003): BIGINT value is out of range` for TiDB engine. However, pushing this expression into TiFlash leads to an overflowed result `2` without any error message.
+> - Window Function
```
Comment (tisonkun) [inline tiflash/use-tiflash.md:290]:
```suggestion
> - Window Function.
```

File: tiflash/use-tiflash.md [hunk 3]
```diff
@@ -277,11 +277,20 @@ explain select count(*) from customer c join nation n on c.c_nationkey=n.n_natio
...
+> - Reading data from TiKV
```
Comment (tisonkun) [inline tiflash/use-tiflash.md:291]:
```suggestion
> - Reading data from TiKV.
```

File: tiflash/use-tiflash.md [hunk 4]
```diff
@@ -277,11 +277,20 @@ explain select count(*) from customer c join nation n on c.c_nationkey=n.n_natio
...
+>
+> In addition, in MPP mode TiFlash does not support:
+>
+> - Partition table. For any query on a partitioned table, MPP mode is triggered.
```
Comment (tisonkun) [inline tiflash/use-tiflash.md:295]: is not triggered?

**Comment by TomShawn** [comment]
Confirmed with @ilovesoup that https://github.com/pingcap/docs-cn/pull/5902 's change is moved to https://github.com/pingcap/docs-cn/pull/5891. Therefore, this PR can be replaced by the aligned PR of #5902.

**Pull request closed without merging**